src/persistence/Elsa.Persistence.EntityFrameworkCore/DbContexts/SqlServerContextFactory.cs

```diff
@@ -17,7 +17,10 @@ public SqlServerContext CreateDbContext(string[] args)
             
             optionsBuilder.UseSqlServer(
                 connectionString,
-                x => x.MigrationsAssembly(migrationAssembly)
+                x =>
```

Comment:
Any reasons as to why you changed from lambda expression to body statements?

Reply:
This has been removed and set back to the way that it was.